app/Views/errors/html/error_404.php

```diff
@@ -70,13 +70,13 @@
 </head>
 <body>
     <div class="wrap">
-        <h1>404 - File Not Found</h1>
+        <h1>404</h1>
 
         <p>
             <?php if (! empty($message) && $message !== '(null)') : ?>
                 <?= nl2br(esc($message)) ?>
             <?php else : ?>
-                Sorry! Cannot seem to find the page you were looking for.
+                <?= lang('Errors.404sorryCannotFind') ?>
```

Comment:
Is there any way to display this message?

Reply:
If any falsey string is passed into the PageNotFoundException it would hit this (only explicit `null` uses the default lang message). Also if someone threw the exception directly it would have an empty string for the message: https://3v4l.org/FDkpg

What I want to know is when it would ever be `'(null)'`; that seems oddly specific and also never should happen.

Reply:
`'(null)'` is there:
https://github.com/codeigniter4/CodeIgniter4/blob/0e70ccd62e6e512e454b63aaf15ab7488e508f47/system/Debug/Exceptions.php#L272
But `$exception->getMessage()` never returns null.

Reply:
Yeah that's a mistake. I'm surprised that hasn't been caught by static analysis - very clear not-nullable logic error. I don't think `'(null)'` is ever helpful or needs accounting for, we should remove it.

Reply:
It is suppressed.
```
		-
			message: "#^Expression on left side of \\?\\? is not nullable\\.$#"
			count: 1
			path: system/Debug/Exceptions.php
```